vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.css

```diff
@@ -1,38 +1,70 @@
 .vuuLayoutList {
+    --vuuLayoutList-width: 90%;
     --vuuMeasuredContainer-flex: 1 1 1px;
     align-self: stretch;
     display: flex;
     flex-direction: column;
     height: 100%;
     overflow: hidden;
+    padding-right: 6px;
+
+    /* Works on Firefox */
+    scrollbar-width: thin;
+    scrollbar-color: #D9D9D9 transparent;
```

Comment:
I would consider removing this and just keeping the default style in Firefox. I don't think it's worth introducing code to change styling when it doesn't actually match the design.

Reply:
I tend to agree with Peter here. If we can't replicate the style from figma or, at least close enough to it, I would leave the default style for firefox

Reply:
[e56647f](https://github.com/ScottLogic/finos-vuu/pull/130/commits/e56647f3b0a52b988744f688ef959a8b07067bf4)